manifests/m/Microsoft/VC++2013Redist-x64/12.0.40660.0/Microsoft.VC++2013Redist-x64.installer.yaml

```
@@ -19,6 +19,8 @@ Installers:
   Scope: machine
   InstallerUrl: https://download.microsoft.com/download/0/5/6/056dcda9-d667-4e27-8001-8a0c6971d6b1/vcredist_x64.exe
   InstallerSha256: 20E2645B7CD5873B1FA3462B99A665AC8D6E14AAE83DED9D875FEA35FFDD7D7E
+  InstallerSwitches:
+    Custom: /norestart
   UpgradeBehavior: install
```

```suggestion
  UpgradeBehavior: uninstallPrevious
```

https://github.com/microsoft/winget-cli/issues/1424#issuecomment-910312191